client/js/shout.js

```diff
@@ -382,6 +386,20 @@ $(function() {
 		}
 	}
 
+	if ($.cookie("highlights") !== null) {
```

Comment:
Use the existing `settings` cookie object, it will be easier to switch to another storage method in the future.